homeassistant/components/mqtt/light/schema_basic.py

```diff
@@ -468,6 +468,12 @@ def brightness_received(msg: ReceiveMessage) -> None:
                 return
 
             device_value = float(payload)
+            if device_value == 0:
+                _LOGGER.warning(
+                    "Brightness cannot be zero in message from '%s'", msg.topic
```

Comment:
```suggestion
                    "Ignoring zero brightness value in message from '%s'", msg.topic
```

Opinion: I prefer to use the kind of message as on line 467